src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -32,6 +33,18 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
     error = '';
     dots = 1;
 
+    readonly firstMessageContent = {
+        textContent: "Hi, I'm Iris! I can help you with your programming exercise. You can know more about me here.",
+        type: IrisMessageContentType.TEXT,
+    } as IrisMessageContent;
+
+    readonly firstMessage = {
+        sender: IrisSender.LLM,
+        id: 1,
+        content: [this.firstMessageContent],
+        sentAt: dayjs(),
+    } as IrisServerMessage;
```

Comment:
Why not only create these objects when you actually need them when sending the first message?